.gitlab-ci.yml

```diff
@@ -1,49 +1,82 @@
-image: ubuntu:22.04
-
 stages:
-  - test
+  - build
+
+.standard-rules:       # Make a hidden job to hold the common rules
+  rules:
+    - if: $CI_PIPELINE_SOURCE == 'merge_request_event'
+    - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH
+  artifacts:
+    paths:
+      - "build/*/*.tar.gz"
 
 .setup_linux: &setup_linux |
+  echo -e "\e[0Ksection_start:`date +%s`:my_setup_linux[collapsed=true]\r\e[0KHeader of the setup linux collapsible section"
   DEBIAN_FRONTEND=noninteractive
 
-  # set time-zone
-  TZ=Canada/Pacific
-  ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+  # XXX WHY? set time-zone
+  # TZ=Canada/Pacific
+  # ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
```

Comment:
As far as I remember, this is required for proper downloading of the tools

Reply:
see my [gitlab pipline](https://gitlab.com/ClausKlein/cpp_vcpkg_project/-/pipelines/984127431)